packages/web-runtime/src/services/auth/userManager.ts

```diff
@@ -212,12 +212,13 @@ export class UserManager extends OidcUserManager {
       email: login?.email || user?.email || '',
       groups: userGroups,
       role,
-      language: login?.language,
-      ...(user.quota &&
-        user.quota.definition !== 'default' &&
-        user.quota.definition !== 'none' && { quota: user.quota })
```

Comment:
I don't get this conditions, we load the user quota on other places without those conditions

Reply:
Me neither 😞